.env.example

```diff
@@ -0,0 +1,17 @@
+ADMIN_USER=admin
+ADMIN_PASSWORD=secret
+ADMIN_EMAIL=admin@example.com
+WEB_PORT=8000
+
```

Comment:
Think this file is missing the deploy .env vars ya?

Reply:
yeah, had those separate, but good point... will combine the files and update the reference in the deploy script